numpy/random/tests/test_random.py

```diff
@@ -352,40 +356,41 @@ def test_bytes(self):
         np.testing.assert_equal(actual, desired)
 
     def test_shuffle(self):
```

Comment:
I don't see any context using either `--pdb-failure` or `--ipdb-failure` (not that I knew about these options, thanks for pointing them out).